independent-publisher-connectors/rSpaceX/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#004F84",

```

Comment:
@troystaylor  Can you change it default iconBrandColor for all the IP connectors?

"iconBrandColor": "#da3b01"

Reply:
Fixed